apps/omg/lib/omg/ethereum_client_monitor.ex

```diff
@@ -42,7 +42,7 @@ defmodule OMG.EthereumClientMonitor do
   defstruct interval: @default_interval,
             tref: nil,
             alarm_module: nil,
-            raised: true,
+            raised: false,
```

Comment:
🤔 Do we have tests to cover this change?

Reply:
same tests should continue working, but I can add more